### PR TITLE
done 3:37

### PR DESCRIPTION
**Pull request by SBrandon0432**
image

done at 3:37

only form that needed a custom validate was name as the rest have built-in types from input

### CODE REVIEW

**Review by SBrandon0432 — COMMENTED** [review]
looks good

File: src/Components/SelfInterviewTwo/ProfileInputs.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            </label>
+
+            <label>
+                Enter Age:
+                <input
+                    type="number"
+                    value={age}
+                    onChange={(e) => {
+                        const age = parseInt(e.target.value);
+                        console.log(typeof age);
```
Comment (CommanderHamster) [inline src/Components/SelfInterviewTwo/ProfileInputs.tsx:37]: You left debug stuff behind

File: src/Components/SelfInterviewTwo/ProfileInputs.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                            DOB,
+                        });
+                    }}
+                />
+            </label>
+
+            <label>
+                About:
+                <textarea
+                    required={true}
```
Comment (CommanderHamster) [inline src/Components/SelfInterviewTwo/ProfileInputs.tsx:65]: Does this work?

File: src/Components/SelfInterviewTwo/ProfileInputs.tsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+                    value={DOB}
+                    onChange={(e) => {
+                        const DOB = e.target.value;
+                        console.log(typeof DOB);
+                        setFormFieldInputs({
+                            ...formFieldInputs,
+                            DOB,
+                        });
+                    }}
+                />
```
Comment (CommanderHamster) [inline src/Components/SelfInterviewTwo/ProfileInputs.tsx:59]: Where is the validation for this field. It must be a date in the past

**Review by CommanderHamster — COMMENTED** [review]
This one feel a little sloppy to me. React code is fine, but we missed the date validation from the prompt. Since you said you finished in 37 min, I would have liked to see more time spent on cleaning up the code, validating the prompt and styling. Styling being the last thing, but if you had the time.

**Pull request closed without merging**